fellow-submissions/peter-rose.md

```diff
@@ -0,0 +1,148 @@
+   in this .md file for a good example of this.
+   
+   Thirdly, var can be accessible anywhere in function but let and const can only be accessible inside the block where you first declare them.
+
+**2. What will the following code log? Why?**
+    ```javascript
+    let a = 'outer';
+
+    function testScope() {
+      let a = 'inner';
+      console.log(a);
+    }
+
+    console.log(a);
+    testScope();
+    console.log(a);
+    ```
+    <!---->
+    console.log(a) will ALWAYS log as outer. The function called on line 15 is refering to the variable 'a' thats initialized within itself
+    so it'll always refer to that value which is inner.
+    //ouput    outer --> inner --> outer
+
+**3. What will the following code log? Why?**
+    ```javascript
+    function hello() {
+      let a = 'hello';
+    }
+
+    hello();
+    console.log(a);
+    ```
+    <!---->
+    Code will result in an error, because a is not defined. this is because let has a block scope so the value of a cannot be 
+    accessed because the "let" keyword is only accessible inside the block where they are declared, 
+    just like "const" and unlike "var".
+
+**4. What does the following code log? Why?**
+    ```javascript
+    console.log(a);
+
+    var a = 1;
+    ```
+    <!---->
+    The code logs as undefined. This is because var may have a normal scope so due to hoisting the console knows a 
+    variable named a does exist but it is called upon because that variable is initialized, and cannot retrieve the value.
+  
+**5. What does the following code log? Why?**
+    ```javascript
+    console.log(a);
+
+    let a = 1;
+    ```
+    <!---->
+    The code results in an error saying that 'a' is not defined. This is signaificantly different from the previous example because
+    let has it's own scope of block.  It cannot be called upon at all due to this. So when ran to the console, it does not run because
+    the console thinks youre asking for something that does not exist.
+
+**6. In the code snippet below, the function declaration comes _after_ the variable declaration in the code below. Why does it still work?**
+    ```javascript
+    let name = "Devonte";
+
+    function sayItLoud() {
+      console.log(`${name}!!!`)
+    }
+
+    sayItLoud();
+    ```
+    <!---->
+    This still works because the variable name is declared and initialized outside of the function and has a global scope allowing it
+    to be called upon at any point our script. Will work with any variable keyword such as var, let & const.
+     
+**7. Why does the following code throw an error? What is the key difference between it and the example above?**
+    ```javascript
+    sayItLoud();
+
+    function sayItLoud() {
+      console.log(`${greeting}!!!}`)
+    }
+
+    let greeting = "Hello"
+    ```
+    <!---->
+    The key difference between this and the previous example is that the function is called upon at the evry start
```

Comment:
You are super close here, Peter. You are right that the issue is that we invoked the function on line 1. However, we know that function declarations are hoisted to the top of their scope. Are variables declared with `let`?